desktop/cmp/panel/Panel.js

```diff
@@ -138,7 +148,17 @@ export class Panel extends Component {
             maskElem = maskProp;
         }
 
```

Comment:
Can we DRY up 3) and 4).  e.g. 
`const maskElem = parseLoadDecorator(maskProp, mask )`
`const loadIndicatorElem = parseLoadDecorator(loadIndicatorProp, loadIndicator)`

Reply:
Done.